backend/app/Http/Controllers/FollowController.php

```diff
@@ -2,9 +2,60 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\Follow;
+use App\Models\User;
 use Illuminate\Http\Request;
 
+use function PHPUnit\Framework\isEmpty;
+
 class FollowController extends Controller
 {
-    //
+    public function index()
+    {
+        return $this->showAll(Follow::all());
+    }
+
+    public function store()
+    {
+        $data = request()->validate(
+            [
+                'follower' => ['required', 'integer'],
+                'following' => ['required', 'integer']
+            ]
+        );
+
+        // Check if already following
+        $checkDuplicate = Follow::where('follower_id', $data['follower'])->where('following_id', $data['following'])->get();
```

Comment:
This can be shorten, you can group `where()` in laravel.
related topic: https://laravel.com/docs/7.x/queries#where-clauses
```suggestion
        $checkDuplicate = Follow::where([
            ['column_1', '=', $data['column_1]],
            ['column_2', '=', $data['column_2]],
        ])->get();
```